self_hosting_machinery/finetune/configuration/supported_models.py

```diff
@@ -1,7 +1,7 @@
 __all__ = ['config']
 
 _fim_train_ds_pipeline = {
-    "ds_opts": "n_ctx={n_ctx},debug=0,seed=42,shuffle_depth=256,"
+    "ds_opts": "n_ctx={n_ctx},debug=0,shuffle_depth=256,"
```

Comment:
we do not need fixed seed, 'cause it makes data per epochs the same
while multigpu training though it is necessary, so related changes would be in the next PR